### DISCUSSION TITLE
Tesflight Missing certificates in app store.

### DISCUSSION THREAD

**Discussion by darbaoui** [fastlane/fastlane#18281] [Q&A]
After success to build the app and upload it to apple testflight using (github actions and fastlane)  i can't receive any notification about this build from apple.
the problem is i need each time to manage a Missing certificates from it's interface.

<img width="492" alt="screenshot_2021-02-25_at_14 18 07" src="https://user-images.githubusercontent.com/6907747/109160283-e01fee00-7775-11eb-940d-9ac9a98a9ff6.png">

**Comment by dinsen** [comment]
Hi @darbaoui 
Can you provide more information? 
I have never seen that warning before.

**Comment by narcdev** [comment]
Hi,

Any news on this issue ? I have the same problems.

Thanks in advance.

**Reply by dinsen** [reply]
It will be pretty hard to give any help if we don't get more information.

**Reply by narcdev** [reply]
My bad, I think I have found how to solve it. 

It's with the "uses_non_exempt_encryption" parameter.

I try and let you know

**Reply by narcdev** [reply]
Ok it was that. You can close the issue !
